kanidm_book/src/administrivia.md

```diff
@@ -46,7 +50,8 @@ This is a simple backup of the data volume.
 Automatic backups can be generated online by a `kanidmd server` instance
 by including the `[online_backup]` section in the `server.toml`.
 This allows you to run regular backups, defined by a cron schedule, and maintain
-the number of backup versions to keep. An example is located in [examples/server.toml](../../examples/server.toml).
+the number of backup versions to keep. An example is located in 
+TODO this link is 404 [examples/server.toml](../../examples/server.toml).
```

Comment:
This is probably best to be https://github.com/kanidm/kanidm/blob/master/examples/server.toml